### DISCUSSION TITLE
Calculating hights of manholes, in pipe and out pipes in QGEP

### DISCUSSION THREAD

**Discussion by PeterZamudio** [QGEP/QGEP#769] [Ideas]
Hello everyone.

I would like to share my solution for Calculating hights of manholes, in pipe and out pipes in QGEP as we agreed in the last TEKSI Technical Goup Meeting.

I have made a video with a small demonstration. I hope you like it and look forward to your feedback.

https://app.box.com/s/mxowe0xr3p2qvvk4ii0hlkncidxq0njz
The video was to big to attach here so I have made a download link.

Peter Zamudio
geopoint lütolf ag

**Comment by sjib** [comment]
@PeterZamudio Thanks a lot for the video. Did you ever think of having the Schachtprotokoll digitally instead of paper? Then we could create a qfield project with the E1, E2, A1, A2 etc. to directly fill in and do the calculation of the levels automatically on import?

@varrieta Does your qfield tool support this use case?

**Reply by varrieta** [reply]
For most of it yes !  #736
(I'm quite late with the doc)

**Comment by PeterZamudio** [comment]
@sjib Yes we have discussed that, and the problem is mainly the direct import of GPS measurements into the GIS point for point. With the new slanted GPS pole it would be interesting. But in effect we always end up back with the protocol on paper as that is still needed and the most efficient and fastest way in the field. I just redid a whole area in town and the work after in the office was much easier on two big screens, a mouse and keyboard rather than a pen and small tablet in the field. But we are open to new ideas :-)
